ui/pages/confirm-transaction-base/low-gas-warning/low-gas-warning.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import ActionableMessage from '../../../components/ui/actionable-message/actionable-message';
+import { useGasFeeContext } from '../../../contexts/gasFee';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const LowGasWarning = () => {
```

Comment:
Same thing as the comment above, what do you think about LowPriorityMessage or LowPriorityExplanation? I know that this _appears_ as a warning, so I can see why you might have put "warning" in the name, so if you still like that then perhaps LowPriorityWarning?

Reply:
Also renamed